state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -399,7 +400,7 @@ fn re_request_threshold_signature_on_invalid_tx_params() {
 
 			assert_eq!(
 				MockThresholdSigner::<MockEthereumChainCrypto, RuntimeCall>::signature_result(0),
-				AsyncResult::Void
+				(Default::default(), AsyncResult::Void)
```

Comment:
Think we should use some non-default const for these instead of Default. Using Default will very easily hide potential bugs (e.g. if we unwrap_or_default() and hit a default, we could still pass the test)

Reply:
We normally use Default for things that we dont test for in the test so I dont think its a big problem. In any case, I tried to do this but it is not so simple since signature_result is part of a mock pallet. It can be done but would take a bit of time. Dont know if we want this as part of this PR.